docs/content/develop/make-a-breaking-change.md

```diff
@@ -181,6 +181,21 @@ Example (`google_container_cluster.enable_shielded_nodes`)
 * https://github.com/GoogleCloudPlatform/magic-modules/pull/5263 changed the default value for the field and added an upgrade guide entry
 * Note: In 4.0.0 the upgrade guide was split between main and the major release branch. Going forward, those changes will be made against main exclusively.
 
+## Contributing to the next major release (`5.0.0`)
+
+For the next major release, `5.0.0`, a specific branch named [`FEATURE-BRANCH-major-release-5.0.0`](https://github.com/GoogleCloudPlatform/magic-modules/tree/FEATURE-BRANCH-major-release-5.0.0) will be managed parallel to `main` in the magic-modules repo.
+All breaking changes targeting `5.0.0` must be committed to `FEATURE-BRANCH-major-release-5.0.0`.
```

Comment:
We should cover how folks that made a change in `main` and now need to follow it up in `5.0.0` should do so. Naively, they'd need to replay those changes, possibly introducing a merge conflict. Should we encourage them to wait until a branch sync? If so, can we state how often they'll sync?